instrument/sampleenvironments/ISIS/FlatPlateFrame.xml

```diff
@@ -0,0 +1,52 @@
+<!-- Definition of FlatPlate-->
```

Comment:
Rest of the code looks good (in particular running the fit all stuff in an async task and tidying up the observers and putting them together in the fitting presenter) - just this one file that I think wasn't meant to be included?
About to give it a test!

Reply:
I've removed it now, great spot!